Generator/Sources/needle/GenerateCommand.swift

```diff
@@ -34,13 +34,15 @@ class GenerateCommand: Command {
     private let suffixes: OptionArgument<[String]>
     private let additionalImports: OptionArgument<[String]>
     private let scanPlugins: OptionArgument<Bool>
+    private let singleThreaded: OptionArgument<Bool>
```

Comment:
Do we want to enable this from the command line? When we debug, we'll be running through Xcode right?

Reply:
Moved to env flag.